api/base/filters.py

```diff
@@ -51,7 +51,10 @@ class OSFOrderingFilter(OrderingFilter):
     # override
     def filter_queryset(self, request, queryset, view):
         ordering = self.get_ordering(request, queryset, view)
-        if isinstance(queryset, DjangoQuerySet):
+        field_order = self.check_serializer_fields(queryset, request.query_params, view, ordering)
```

Comment:
Is `request.query_params` passed and later used as opposed to using `ordering` because `ordering` has some type of field validation (https://github.com/encode/django-rest-framework/blob/master/rest_framework/filters.py#L193)?